### PR TITLE
add new package: daemonize

### PR DESCRIPTION
**Pull request by vmiheer**

### CODE REVIEW

File: var/spack/repos/builtin/packages/daemonize/package.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import os
+import sys
+
+from spack.package import *
+
+
+class Daemonize(Package):
+    """The Emacs programmable text editor."""
+
+    homepage = "http://software.clapper.org/daemonize"
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/daemonize/package.py]: Please switch the protocol here to `https`.

**Review by tldahlgren — COMMENTED** [review]
Confirmed the homepage, branch, and version sha256.  Successfully built the package as well.  

Would prefer use of `https` protocol for the homepage.

**Review by tldahlgren — DISMISSED** [review]

**Comment by tldahlgren** [comment]
Looks like there are a few changes suggested by the checks:

```
==> Running flake8 checks
var/spack/repos/builtin/packages/daemonize/package.py:6: [F401] 'os' imported but unused
var/spack/repos/builtin/packages/daemonize/package.py:7: [F401] 'sys' imported but unused
  flake8 found errors
```

**Comment by vmiheer** [comment]
My bad! Copy paste error.

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**